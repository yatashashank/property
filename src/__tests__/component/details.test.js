import React from 'react';
import toJson from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import { shallow,render,mount } from 'enzyme';

import Details from '../../Components/Property/details'


describe('<Details />', () => {
//console.log('details  ',Details)
    let wrapper;
    beforeEach(() => {
      wrapper = shallow(<Details />);
    })

  it('snapshot testing for Details component', () => {
    //wrapper = shallow(<Details />);
    let wrap=wrapper.first().shallow()
    expect(toJson(wrap)).toMatchSnapshot()
  });

  it('root class name check', () => {
    //const wrapper = shallow(<Details />);
    const tree=wrapper.state();
    console.log('tree ',)
    //expect(tree.length).toBe(1)
  });
})