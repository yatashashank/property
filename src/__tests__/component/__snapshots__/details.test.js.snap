// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Details /> snapshot testing for Details component 1`] = `
<Route>
  <Component />
</Route>
`;
